core/src/main/java/org/elasticsearch/action/delete/TransportDeleteAction.java

```diff
@@ -48,72 +42,48 @@
 /**
  * Performs the delete operation.
  */
-public class TransportDeleteAction extends TransportWriteAction<DeleteRequest, DeleteRequest,DeleteResponse> {
+public class TransportDeleteAction extends TransportWriteAction<DeleteRequest, DeleteRequest, DeleteResponse> {
```

Comment:
wondering - should we mark this as deprecated, so we want use internally? not sure how much it buys us.